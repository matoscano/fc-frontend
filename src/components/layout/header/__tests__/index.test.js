import React from "react";
import { render } from "@testing-library/react";
import Header from "../index";

describe("Header", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
