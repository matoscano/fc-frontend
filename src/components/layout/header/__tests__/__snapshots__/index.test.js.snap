// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<DocumentFragment>
  <header
    class="sc-AxiKw irtYfQ"
  >
    <div
      class="sc-AxhCb jzhRtr"
    >
      <div
        alt="FilmChain logo"
        aria-label="FilmChain logo link"
        class="sc-Axmtr bUgpto"
        to="/"
      >
        <svg
          viewBox="0 0 239 67"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="none"
            fill-rule="evenodd"
            transform="translate(-100.000000, -65.000000) translate(100.000000, 60.000000) translate(0.000000, 5.000000)"
          >
            <path
              d="M19.8 10.781c-3.905 3.834-6.094 9.043-6.08 14.472 0 .346 0 .693.025 1.023l2.439-2.393a26.015 26.015 0 015.28-4.016 13.814 13.814 0 013.04-4.44l9.57-9.433a3.253 3.253 0 000-4.645 3.375 3.375 0 00-4.708 0L19.8 10.781zm4.625 21.211c4.768-4.67 12.22-5.389 17.82-1.716l4.771-4.693c-8.267-6.223-19.938-5.462-27.299 1.78L1.421 45.393a3.28 3.28 0 00-.985 2.338c0 .878.354 1.719.985 2.339a3.359 3.359 0 004.67 0l18.334-18.078zM49.067 56.29c4.15-4.092 6.346-9.722 6.044-15.495l-2.429 2.393a26.028 26.028 0 01-5.295 4.016 13.811 13.811 0 01-3.029 4.44l-9.57 9.433a3.256 3.256 0 000 4.645 3.376 3.376 0 004.709 0l9.57-9.432zm-4.626-21.211c-3.24 3.17-7.832 4.612-12.339 3.874v-.016a14.435 14.435 0 01-5.482-2.142l-4.77 4.693a20.82 20.82 0 009.139 3.92c1.143.202 2.301.302 3.462.3 5.515 0 10.803-2.158 14.698-6l18.316-18.046a3.253 3.253 0 000-4.645 3.351 3.351 0 00-2.358-.954 3.35 3.35 0 00-2.352.97L44.441 35.079zm-6.24-1.543c-.001 2.051-1.692 3.713-3.776 3.713-2.085 0-3.774-1.664-3.774-3.715 0-2.052 1.69-3.715 3.775-3.715a3.811 3.811 0 012.675 1.086 3.69 3.69 0 011.11 2.63h-.01z"
              fill="#DA6D6A"
            />
            <path
              d="M81.767 45.692V21.38h13.121v4.567h-7.986v4.535h9.28v4.535h-9.28v10.676h-5.135zm16.669-22.03a2.72 2.72 0 011.663-2.574 2.8 2.8 0 013.047.561 2.691 2.691 0 01.606 2.992 2.769 2.769 0 01-2.596 1.667c-1.482-.02-2.683-1.188-2.72-2.646zm5.101 22.03h-4.7V29.205h4.705v16.487h-.005zm8.02-24.313v24.313h-4.706V21.38h4.706zm19.649 24.313H126.5v-8.22c0-2.834-1.49-3.984-3.61-3.984-2.007.248-3.464 2-3.314 3.984v8.22h-4.705V29.205h4.705v1.15a6.001 6.001 0 014.107-1.496 6.91 6.91 0 015.667 2.598 7.523 7.523 0 015.963-2.598c5.137 0 7.622 3.842 7.622 8.613v8.22h-4.672v-8.22a3.505 3.505 0 00-.83-2.739 3.623 3.623 0 00-2.616-1.245 3.84 3.84 0 00-2.65 1.25 3.719 3.719 0 00-.961 2.734v8.22zm39.895-10.74c-.767 6.35-6.225 11.14-12.723 11.165-6.143-.014-11.408-4.326-12.537-10.268-1.13-5.943 2.196-11.836 7.919-14.034 5.722-2.198 12.225-.08 15.485 5.044l-4.008 2.803c-1.472-2.351-4.053-3.809-6.859-3.874-4.176-.106-7.707 3.021-8.038 7.12-.331 4.098 2.653 7.735 6.794 8.28 4.141.544 7.992-2.195 8.765-6.235h5.202zm19.85 10.74h-4.672v-8.22c0-2.645-1.557-3.984-3.479-3.984-2.224.014-4.017 1.796-4.01 3.984v8.22h-4.704V21.38h4.704v9.007a6.177 6.177 0 014.409-1.527c4.571 0 7.752 3.842 7.752 8.613v8.22zm19.385-16.487v16.487h-4.705v-1.165a5.623 5.623 0 01-4.075 1.527c-4.62-.275-8.224-4.042-8.224-8.597s3.604-8.323 8.224-8.598a6.428 6.428 0 014.075 1.496v-1.15h4.705zm-4.506 8.267a3.719 3.719 0 00-1.03-2.802 3.844 3.844 0 00-2.78-1.182c-2.173.085-3.89 1.844-3.89 3.984 0 2.14 1.717 3.9 3.89 3.984a3.825 3.825 0 002.787-1.175 3.7 3.7 0 001.023-2.809zm7.423-13.81a2.722 2.722 0 011.664-2.576 2.801 2.801 0 013.05.56 2.693 2.693 0 01.605 2.994 2.77 2.77 0 01-2.599 1.668c-1.483-.018-2.686-1.186-2.723-2.646h.003zm5.103 22.03h-4.705V29.205h4.705v16.487zm20.18 0h-4.673v-8.22c0-2.645-1.557-3.984-3.478-3.984a3.874 3.874 0 00-2.907 1.115 3.746 3.746 0 00-1.107 2.87v8.22h-4.704V29.204h4.704v1.15a6.356 6.356 0 014.409-1.496c4.571 0 7.753 3.842 7.753 8.613v8.22h.003z"
              fill="#FFFFFF"
            />
          </g>
        </svg>
      </div>
      <div
        class="sc-AxhUy hBOCMP"
        data-testid="toggle"
      >
        <div
          class="sc-AxgMl smKsC"
        />
      </div>
      <div
        class="sc-AxheI cQzKeB"
      >
        <div
          class="sc-AxjAm jkyjjb"
        >
          <a
            class="sc-AxirZ bZQjhD"
            href="/wallet"
          >
            Wallet
          </a>
        </div>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
